app/decorators/middleware_domain_decorator.rb

```diff
@@ -0,0 +1,13 @@
+class MiddlewareDomainDecorator < Draper::Decorator
+  delegate_all
+  include MiddlewareDecoratorMixin
+
+  def fonticon
+    'fa fa-exchange'.freeze
```

Comment:
@serenamarie125  @catrobston we'll need 2 new icons - for middleware_domain and middleware server group.
cc @epwinchell (please also note the png placeholders)